frontend/src/components/SearchArtist.js

```diff
@@ -0,0 +1,43 @@
+import styled from 'styled-components/macro'
+import { useState } from 'react'
+import { Link } from 'react-router-dom'
+
+export default function SearchArtist() {
+  const [artistSearchTerm, setArtistSearchTerm] = useState('')
+
+  return (
+    <SearchArtistContainer>
+      <form>
+        <input
+          placeholder="the artist I want to find out about is..."
+          type="text"
+          value={artistSearchTerm}
+          onChange={({ target }) => setArtistSearchTerm(target.value)}
+        />
+        <Link to={`/artistsearch/${artistSearchTerm}`}>
+          <button>SEARCH</button>
+        </Link>
```

Comment:
instead of including a button between your `Link` you can also use a `styled.Link` and style it like a button to reduce your code

Reply:
I will keep it like this for now. I will try the styled.Link next time I have to frontend work ...